src/block/CaveVines.php

```diff
@@ -109,7 +109,7 @@ public function onNearbyBlockChange() : void{
 	}
 
 	public function place(BlockTransaction $tx, Item $item, Block $blockReplace, Block $blockClicked, int $face, Vector3 $clickVector, ?Player $player = null) : bool{
-		if(!$this->canBeSupportedBy($blockReplace->getSide(Facing::UP))){
+		if(!$this->canBeSupportedBy($blockReplace->getSide(Facing::UP)) || $face !== Facing::DOWN){
```

Comment:
I don't think this should be necessary? usually blocks can figure out where they are supposed to go regardless of which face was clicked

Reply:
Usually yes, but in Bedrock, cave vines can be placed only if facing is down.

Reply:
I think that's an unnecessary restriction. There's no real reason to enforce it in PM as far as I can see.

Reply:
I mean that without this check the vines can be placed when client thinks it can't.  I send a video:

Without face check: https://www.youtube.com/watch?v=FsZUy4TqthE
With face check: https://youtu.be/7Fc_yt_Trr0

Also, this behavior is only unique for cave vines - neither weeping vines nor hanging roots have this placement behavior.

Reply:
Yes, I understand the problem. I'm just saying I don't think it's necessary to restrict it in PM.